provider/internal/builtin/service_client.go

```diff
@@ -324,15 +324,19 @@ func (b *builtinServiceClient) getLocation(ctx context.Context, path, content st
 		line := strings.Trim(part, " ")
 		line = strings.ReplaceAll(line, "\t", "")
 		line = regexp.QuoteMeta(line)
-		if line != "" {
-			lines = append(lines, line)
-		}
+		lines = append(lines, line)
+	}
+	// remove leading and trailing empty lines
+	if len(lines) > 0 && lines[0] == "" {
```

Comment:
Is there a time/place where you can have content that is 5 empty lines?

Reply:
@shawn-hurley content comes from innerText returned by the XML search, and it depends on the xml query, so theoritically it's possible if someone is intentionally / mistakenly looking for empty lines. in that case, we will end up showing empty lines as code snip

Reply:
What if as we loop, as long as one thing has content, then we use it, if all don't then we skip it?